pkg/types/ovirt/defaults/machinepools.go

```diff
@@ -0,0 +1,34 @@
+package defaults
+
+import (
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/ovirt"
+)
+
+func setMachinePool(p *types.MachinePool) {
+	if p.Platform.Ovirt == nil {
+		p.Platform.Ovirt = &ovirt.MachinePool{}
+	}
+}
+
+func setDefaultAffinityGroups(p *ovirt.Platform, mp *types.MachinePool, agName string) {
+	if mp.Platform.Ovirt.AffinityGroupsNames == nil {
```

Comment:
This field is `omitempty`, so we need to treat nil and an empty slice identically.
```suggestion
	if len(mp.Platform.Ovirt.AffinityGroupsNames) == 0 {
```

Reply:
Done